examples/submittable.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetsTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
+import { Keyring } from '@polkadot/keyring';
+import { cryptoWaitReady } from '@polkadot/util-crypto';
+
+import { AssetsTransferApi, constructApiPromise } from '../src';
+import { TxResult } from '../src/types';
+
+/**
+ * In this example, we are creating a `SubmittableExtrinsic` and showing how one may sign and send it over
+ * a network. Since it is the `SubmittableExtrinsic`, there are a plethora of attached methods to use such as:
+ *
+ * `sign`, `signAsync`, `dryRun`, `addSignature`, `paymentInfo`, etc.
+ */
+const main = async () => {
+	const { api, specName, safeXcmVersion } = await constructApiPromise(
+		'ws://127.0.0.1:9944'
+	);
+	const assetApi = new AssetsTransferApi(api, specName, safeXcmVersion);
+
+	// When declaring this type it will ensure that the inputted `format` matches it or the type checker will error.
+	let callHex: TxResult<'submittable'>;
+	try {
+		callHex = await assetApi.createTransferTransaction(
```

Comment:
did you mean to put callHex here? I see for others you have callInfo

Reply:
Ahh yea this should be callInfo, i'll fix that